pkg/bootstrapper/bootstrapper.go

```diff
@@ -436,15 +436,15 @@ func (wmcb *winNodeBootstrapper) Run() error {
 	}
 	err = wmcb.initializeKubelet()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to initialize kubelet with %v", err)
```

Comment:
Please make this `failed to initialize kubelet: %v `. Change the others in the same manner.

Reply:
:+1: otherwise lgtm